src/components/CategoryBadge/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Data } from '@/pages/api/getMenu';
+import CategoryBadge from '@/components/CategoryBadge/CategoryBadge';
+
+import styles from './CategoryBadge.module.css';
+
+type Props = { menu: Data };
+
+export default function BadgeList({ menu }: Props): JSX.Element {
```

Comment:
⚠️ exporting BadgeList component from CategoryBadge component root file (index.tsx) leads to misunderstanding 

Needs to be refactored:
- move BadgeList component to `components/BadgeList/index.tsx`
- move CategoryBadge component to `index.tsx`